templates/github/.github/workflows/scripts/install.sh.j2

```diff
@@ -204,9 +204,17 @@ fi
 {%- endif %}
 
 {%- if test_gcp %}
-
+cat >> credentials.json << GCP_JSON
+{
```

Comment:
just to understand this better, where this is taken from?

Reply:
from:
https://cloud.google.com/sdk/docs/install
https://cloud.google.com/docs/authentication/provide-credentials-adc#local-dev

Reply:
I pulled in these changes into my pr and the issue seems to persist https://github.com/pulp/pulp_container/pull/1196

Reply:
it seems it needs more hacks:
https://github.com/fsouza/fake-gcs-server#using-with-signed-urls